ramiro_serra/notes.md

```diff
@@ -0,0 +1,113 @@
+# Trainee program Week#2 notes
+
+After running the docker container, building with catkin, and launching `roslaunch ca_gazebo create_empty_house.launch`, we see gazebo launching and spawning a robot.
+
+We can list the topics with `rostopic list`, only showing some:
+
+    ...
+    /create1/cmd_vel
+    /create1/laser/scan
+    /create1/mobile_base/sensors/bumper_pointcloud
+    /create1/move_base/current_goal
+    /create1/move_base/goal
+    /create1/move_base/local_costmap/costmap
+    /create1/move_base/local_costmap/costmap_updates
+    /create1/move_base_simple/goal
+    /create1/nodelet_manager/bond
+    /create1/odom
+    /create2/move_base/local_costmap/costmap_updates
+    ...
+
+Now we launch the teleop script, `roslaunch ca_tools key_teleop.py`, but we can't move the turtle, let's check the `rostopic list` again:
+
+    ...
+    /cmd_vel
+    /create1/amcl/parameter_descriptions
+    /create1/amcl/parameter_updates
+    /create1/amcl_pose
+    /create1/bumper
+    /create1/cliff
+    /create1/cmd_vel
+    /create1/front_left_cliff_sensor/scan
+    ...
+
+We can see there's 2 cmd_vel topics, let's check which one is being published on:
+
+`rostopic echo /cmd_vel`:
+
+    ...
+    linear: 
+    x: 0.0
+    y: 0.0
+    z: 0.0
+    angular: 
+    x: 0.0
+    y: 0.0
+    z: 0.0
+    ---
+    linear: 
+    x: 0.0
+    y: 0.0
+    z: 0.0
+    angular: 
+    x: 0.0
+    y: 0.0
+    z: 0.0
+    ...
+
+`rostopic echo /create1/cmd_vel`:
+
+    **nothing**
+
+We can go even further and check with rqt_graph, let's see.
+![teleop_unaligned](media/teleophanging.png)
```

Comment:
In this graph you're only showing the nodes. With `rqt_graph` you can see the topics too. Show that output instead.